pkgs/top-level/all-packages.nix

```diff
@@ -7796,6 +7800,10 @@ with pkgs;
 
   belle-sip = callPackage ../development/libraries/belle-sip { };
 
+  libbfd = callPackage ../development/libraries/libbfd { };
```

Comment:
`bfd`? `libbfd`? `binutils-bfd`? Help me bikeshed!

Reply:
Maybe rename `binutils-opcodes` to `libopcodes` for consistency with `libbfd`? It seems to be known under this name, e.g. in http://mkfs.github.io/content/opdis/

Reply:
Ah, good precedence finding!